components/automate-cli/pkg/verifyserver/services/systemuserservice/systemuserservice.go

```diff
@@ -0,0 +1,173 @@
+package systemuserservice
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/lib/executil"
+	"github.com/chef/automate/lib/logger"
+	"github.com/chef/automate/lib/userutils"
+)
+
+type SystemUserService interface {
+	GetSystemUserServiceDetails() *models.SystemUserResponse
+}
+
+type SystemUserServiceImp struct {
+	exec executil.ExecCmdService
+	user userutils.UserUtil
+	Log  logger.Logger
+}
+
+func NewSystemUserService(log logger.Logger) *SystemUserServiceImp {
+	return &SystemUserServiceImp{
+		exec: executil.NewExecCmdServiceImp(),
```

Comment:
take these also input for function so if any other service uses this they can provide there on implementation of these utils. also it will be easy to mock them in testing

Reply:
Done